concepts/bools/about.md

```diff
@@ -27,56 +31,68 @@ True
 
 All `boolean operators` are considered lower precedence than Pythons [`comparison operators`][comparisons], such as `==`, `>`, `<`, `is` and `is not`.
 
+
 ## Type Coercion and Truthiness
 
-The bool function ([`bool()`][bool-function]) converts any object to a Boolean value. By default all objects return `True` unless defined to return `False`.
+The `bool` function ([`bool()`][bool-function]) converts any object to a Boolean value.
+ By default all objects return `True` unless defined to return `False`.
 
 A few `built-ins` are always considered `False` by definition:
 
 - the constants `None` and `False`
 - zero of any _numeric type_ (`int`, `float`, `complex`, `decimal`, or `fraction`)
 - empty _sequences_ and _collections_ (`str`, `list`, `set`, `tuple`, `dict`, `range(0)`)
 
+
 ```python
 >>>bool(None)
 False
 
 >>>bool(1)
 True
+
 >>>bool(0)
 False
 
 >>>bool([1,2,3])
 True
+
 >>>bool([])
 False
 
 >>>bool({"Pig" : 1, "Cow": 3})
 True
+
 >>>bool({})
 False
 ```
 
-When a object is used in a _boolean context_, it is evaluated transparently as _truthy_ or _falsey_ using `bool()`:
+When an object is used in a _boolean context_, it is evaluated transparently as _truthy_ or _falsey_ using `bool()`:
```

Comment:
This is one missed "Boolean" capitalization @BethanyG .

Reply:
Thank you for catching that!  Looks like I can't slide that one into this..so I'll do a tiny little PR-let here in a few minutes.